bitnami/jaeger/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+annotations:
+  category: Infrastructure
+apiVersion: v2
+appVersion: 1.39.0
+dependencies:
```

Comment:
I'm missing the Chart.lock file fruit of running `helm dependency update`

Reply:
[fcd86a9](https://github.com/bitnami/charts/pull/13480/commits/fcd86a944101f47839aec088fcafa2ceab39e041)